### PR TITLE
WIP: End to End test for vision and audio

### PR DESCRIPTION
**Pull request by wirthual**
Started working on adding end to end tests for audio and vision. #378 

Most of the test cases cover sending URLs to the endpoints, because based on my observation this is the only supported mode of operation.

I added a test case which shows the behaviour of sending only text named `test_{audio,vision}_single_text_only`. Sending text results in a 422 Status response.

Looking through the code it seems the pydantic layer only accepts URL formated strings: See [here](https://github.com/michaelfeil/infinity/blob/6c1ad6812b27e1be23a113b225c437fe0f843b97/libs/infinity_emb/infinity_emb/fastapi_schemas/pymodels.py#L84)

Even with disabled pydandic checks it seems theres only suport for url or list of urls based on this [code](https://github.com/michaelfeil/infinity/blob/6c1ad6812b27e1be23a113b225c437fe0f843b97/libs/infinity_emb/infinity_emb/infinity_server.py#L362)

### CODE REVIEW

File: docs/docs/cli_v2.md [hunk 1]
```diff
@@ -1,19 +1,14 @@
-# CLI v2 Documentation
```
Comment (michaelfeil) [inline docs/docs/cli_v2.md:1]: weird, this is autogenerated differently. Try running `make precommit` again?

**Comment by michaelfeil** [comment]
Looks good to me!

File: docs/docs/cli_v2.md (hunk 1 above)
Comment (wirthual) [reply docs/docs/cli_v2.md]: Running `make precommit` indicates changes in the markdown file. I checked out the version from your main branch now.

File: libs/infinity_emb/tests/conftest.py [hunk 2]
```diff
@@ -8,8 +8,10 @@
 pytest.DEFAULT_BERT_MODEL = "michaelfeil/bge-small-en-v1.5"
 pytest.DEFAULT_RERANKER_MODEL = "mixedbread-ai/mxbai-rerank-xsmall-v1"
 pytest.DEFAULT_CLASSIFIER_MODEL = "SamLowe/roberta-base-go_emotions"
+pytest.DEFAULT_AUDIO_MODEL = "laion/clap-htsat-unfused"
+pytest.DEFAULT_VISION_MODEL = "wkcn/TinyCLIP-ViT-8M-16-Text-3M-YFCC15M"
 
-pytest.ENGINE_METHODS = ["embed", "image_embed", "classify", "rerank"]
+pytest.ENGINE_METHODS = ["embed", "image_embed", "classify", "rerank", "audio_embed"]
```
Comment (michaelfeil) [inline libs/infinity_emb/tests/conftest.py:14]: nice spot

**Review by michaelfeil — APPROVED** [review]
lgtm

**Pull request merged**